app/src/main/java/br/org/cesar/knot_setup_app/fragment/configureGatewayWifi/ConfigureGatewayWifiContract.java

```diff
@@ -0,0 +1,15 @@
+package br.org.cesar.knot_setup_app.fragment.configureGatewayWifi;
+
+public interface ConfigureGatewayWifiContract {
+
+    interface ViewModel {
+        void setWifiSSID(String wifiName);
+        void onDisconnected();
+        void onWriteFailed();
+    }
+
+    interface Presenter {
+        void onFocus();
+        void onSetWifiClicked(String ssid, String pssd);
```

Comment:
if `pssd` means password, either use the entire word or keep the same pattern as the previous commit (I think it's pwd).